spec/system/planning_applications/validating/fee_items_validation_spec.rb

```diff
@@ -440,31 +454,43 @@
 
         click_link "Check fee"
 
-        expect(page).to have_content("Check the response to fee change request")
+        expect(page).to have_content("Check applicant response and update fee paid")
         expect(page).to have_content("Officer request")
 
         inset_texts = page.all(".govuk-inset-text")
         within(inset_texts[0]) do
-          expect(page).to have_content("Reason it is invalid: Incorrect fee")
-          expect(page).to have_content("How it can be made valid: You need to pay a different fee")
+          expect(page).to have_content("Reason fee is invalid:")
+          expect(page).to have_content("Incorrect fee")
+          expect(page).to have_content("What the applicant needs to do")
+          expect(page).to have_content("You need to pay a different fee")
           expect(page).to have_content(closed_other_change_validation_request.updated_at.to_fs)
         end
 
+        expect(page).to have_content("Documents provided by applicant")
+        within(".govuk-table__body") do
+          expect(page).to have_link("View in new window")
+          expect(page).to have_content("File name: proposed-floorplan.png")
+          expect(page).to have_content("Date received: 1 January 2021")
+          expect(page).to have_content("This document was uploaded by the applicant")
+        end
+
         expect(page).to have_content("Applicant response")
         within(inset_texts[1]) do
           expect(page).to have_content("I agree with the fee")
           expect(page).to have_content(closed_other_change_validation_request.updated_at.to_fs)
         end
 
-        expect(page).to have_content("Total fee paid")
+        expect(page).to have_content("Confirm total fee paid")
         expect(page).to have_content("Check any extra fee has been received and update the total fee now paid.")
+        expect(page).to have_content("check that the correct fee has been received")
+        expect(page).to have_content("update the total fee paid")
         expect(page).to have_field("planning_application[payment_amount]", with: "100.00")
 
         fill_in "planning_application[payment_amount]", with: "350.22"
         click_button("Continue")
 
         # Display fee item table
-        within(".govuk-table") do
+        within(".fee-table") do
```

Comment:
This actually makes some of my tests easier. I'd solved it with `.govuk-table:nth-of-type(1)`, but this is neater I think.